src/App/Pages/Generator/GeneratorPage.xaml

```diff
@@ -166,6 +166,7 @@
                         <controls:ExtendedSlider
                             DragCompleted="LengthSlider_DragCompleted"
                             Value="{Binding Length}"
+                            AutomationProperties.HelpText="{Binding Length}"
```

Comment:
Maybe a merge vs. rebase? I see Matt's commit in this PR, https://github.com/bitwarden/mobile/pull/827/commits/b2abcda1117044ce6b7a30fc652bb3781b924ee9.

Reply:
Chad is right. I incorrectly rebased a remote branch. Everything should be fine, it will make the history for this commit slightly bloated/repetitive.